travis/test_pylint

```diff
@@ -66,14 +66,17 @@ def get_extra_params(odoo_version):
     return params
 
 
+extra_params_cmd = ['--extra-params', '--load-plugins=pylint_oca']
```

Comment:
Because the plugin disable check pointless-statement in manifest files.
Then if we don't get the version not should show pointless-statement message.